### ISSUE TITLE
Shuffle function can't handle annotation `export` and `icon` correctly

### ISSUE THREAD

**Issue by HowQi** [cherriesandmochi/gdmaim#23]
+ Cross line `@export` is not handled correctly.

Maybe you can merge them into the same line when you encounter a cross-line annotation.

+ `@icon(...)` was wrongly placed in the second line. 

Wrong icon annotation will cause the command line report error, but I don't know if it will affect the actual execution.

Before

``` gdscript
@icon("icon.svg")
extends Node2D

@export
var the_arg1:int
@export
var the_arg2:float
@export
var the_arg3:int
@export var the_arg4:int
var the_arg_n:Vector2

func my_func():
	the_arg1 = 1
	pass

func _ready() -> void:
	my_func()

```

After

``` gdscript
extends Node2D
@icon("icon.svg")
var __凵勹爿丿:Vector2
var the_arg1:int
var the_arg2:float
@export
@export
@export
func __乚殳殳勹():
	the_arg1=1
	pass
func _ready()->void:
	__乚殳殳勹()
@export var __勹乚丶匸:int
var the_arg3:int
```

**Comment by cherriesandmochi** [comment]
> * Cross line `@export` is not handled correctly.
> 
>
> Maybe you can merge them into the same line when you encounter a cross-line annotation.

Oh, ye, should be easy to fix, as `@rpc` annotations work correctly.

> * `@icon(...)` was wrongly placed in the second line.
>
>
> Wrong icon annotation will cause the command line report error, but I don't know if it will affect the actual execution.

Ohh ye, `@icon` should definitely not get shuffled!

Thank you!

**Comment by cherriesandmochi** [comment]
Fixed by 9afb978 and cc8faa3.
Thanks for all the bug reports!

**Comment by HowQi** [comment]
Great! Thank you!
